driver/pelican/main.go

```diff
@@ -233,6 +243,37 @@ func main() {
 				time.Sleep(pollDr)
 			}
 		}()
+
+		occupancy, err := currentPelican.GetOccupancy()
+		if err != nil {
+			fmt.Printf("Failed to retrieve initial occupancy reading: %s\n", err)
+			return
+		}
+		// Start occupancy tracking loop only if thermostat has the necessary sensor
+		if occupancy != types.OCCUPANCY_UNKNOWN {
+			go func() {
+				for {
+					occupancy, err := currentPelican.GetOccupancy()
+					if err != nil {
+						fmt.Printf("Failed to read thermostat occupancy: %s\n", err)
+						done <- true
```

Comment:
Can you change this so that the error is printed out, but the driver doesn't stop?  Until we're more sure of the failure cases of the auxiliary sensors, I think it might make for a more robust deployment if the driver continues to run and try to access the sensors. What do you think?

Reply:
Sure, that makes sense. Just pushed an update to address this.